### DISCUSSION TITLE
Late STT Api response

### DISCUSSION THREAD

**Discussion by AliMaastech** [deepgram/community#962] [General help] [answered]
hi team, i have using deepgram Speech to text api its giving me response for 3 to 4 words within 1 to 2 min. i am facing this issue from last night.

**Comment by jpvajda** [comment, marked as answer]
@AliMaastech  Can you give us more details about your issue? A Request ID and the API Request you are trying to make would be useful. Also what location are you in? Sometimes location can play a big part in latency with any API.
